app/web/src/components/FuncEditor/AttributeBindings.vue

```diff
@@ -217,14 +225,14 @@ const prototypeViews = computed(() =>
 
 const detachFunc = (): FuncAssociations | undefined => {
   if (props.schemaVariantId) {
-    return {
-      ...associations.value,
-      prototypes: associations.value.prototypes.filter(
-        (proto) =>
-          funcStore.schemaVariantIdForAttributePrototype(proto) !==
-          props.schemaVariantId,
-      ),
-    };
+    const prototype = associations.value.prototypes.find(
```

Comment:
When viewing an Attribute Func, if the user clicks the `Detach` button, we need to instead of removing the association from what's in memory and saving the whole thing, call removeBinding.